README.md

```diff
@@ -1 +1,7 @@
-# superx
+superx
```

Comment:
this file was already pushed.
You need to the existing repo so you don't have a conflict.

Reply:
See the discussion in Slack on how to update your local+fork repos with changes landed in head repo